### PR TITLE
docs(radis-0.14): adding installation guide for radis for conda and mamba

### LINKED ISSUE #278: Deploy Radis on Anaconda
### 🎯 Todo

Will be much easier to deal with all dependencies and C-compiled components. 

Issue open to be taken, if anyone is experienced with publishing Conda packages !

### 💡 Possible implementations

- https://levelup.gitconnected.com/publishing-your-python-package-on-conda-and-conda-forge-309a405740cf
- [How to Setup Automatic Uploads to Anaconda from Travis CI in 15 minutes ?](https://gist.github.com/zshaheen/fe76d1507839ed6fbfbccef6b9c13ed9)
- [Building a Conda recipee](https://conda.io/projects/conda/en/latest/user-guide/tasks/use-conda-with-travis-ci.html#building-a-conda-recipe)

---

### PR DESCRIPTION
**Pull request by arunavabasu-03**
Fixes #278 
===============
@erwanp  , now radis is successfully added in anaconda distribution through conda-forge channel .
FeedStock link : https://github.com/conda-forge/radis-feedstock
Pull Request link : https://github.com/conda-forge/staged-recipes/pull/20936

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -21,10 +21,28 @@ Getting Started
...
+
+    conda config --set channel_priority strict 
+
+3.Finally, install radis using the following command::
+
+    conda install radis
+
+or with *mamba*::
+
+    mamba install radis
```
Comment (erwanp) [inline README.rst:44]: do you need the strict priority ? 
Can we do on one line ? 
``` 
conda install radis -c conda-forge
```

**Comment by arunavabasu-03** [comment]
@erwanp , can we merge this ?

**Review by erwanp — APPROVED** [review]
I let you merge

**Comment by erwanp** [comment]
@arunavabasu-03 we will deploy a new RADIS version this weekend, therefore the Conda environment will have to be updated. You're written as the [Feedback maintainer there](https://github.com/conda-forge/radis-feedstock), do you know how to do that ?

**Comment by arunavabasu-03** [comment]
give the release link and commit sha !!

**Pull request merged**